Source/Urho3D/CSharp/Managed/Scene/VertexBuffer.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Urho3DNet
+{
+    public partial class VertexBuffer
+    {
+        public bool SetSize(int vertexCount, VertexMask mask, bool isDynamic)
+        {
+            return SetSize((uint)vertexCount, (uint)mask, isDynamic);
```

Comment:
This is nice. We lack some typesafety in API and int vs uint battle is annoying. This makes C# wrapper a bit more convenient to use :+1:

Reply:
thanks :) ...